src/Views/Form/Templates/Sequoia/assets/css/tributes.scss

```diff
@@ -82,7 +82,32 @@
         }
     }
 
-    input[type='radio']:checked + label::after {
+    // Match style of payment level buttons.
+    input[type='radio']:checked + label.give-tribute-type-button {
+        color: #28C77B !important;
```

Comment:
This looks good! The only thing I would change is to remove the `!important` here and use the form's primary color for text. The primary color is added to the rest of the form in the `$dynamicCss` string here:

https://github.com/impress-org/givewp/blob/develop/src/Views/Form/Templates/Sequoia/Sequoia.php#L128

So you could add a little CSS there to style tribute buttons, then set the text color using `$primaryColor`.

Reply:
Ah, primary color! Good catch 😅 

I pushed an update to use the Dynamic CSS string so that the primary color is incorporated.

![image](https://user-images.githubusercontent.com/10858303/155211819-346c1c2d-d0d7-4869-a05c-786c8ff0f742.png)